src/jit/gentree.h

```diff
@@ -4873,6 +4882,9 @@ struct GenTreePutArgStk : public GenTreeUnOp
     unsigned gtNumSlots;             // Number of slots for the argument to be passed on stack
     unsigned gtNumberReferenceSlots; // Number of reference slots.
     BYTE*    gtGcPtrs;               // gcPointers
+#ifdef _TARGET_ARM_
+    bool gtIsHfa; // isHfa
+#endif
```

Comment:
The call to `IsHfa` involves a JIT/EE interface call, so I think it's reasonable to maintain a flag on the node.